docs/lua-modules/httpserver.md

```diff
@@ -42,14 +46,15 @@ Callback function has 2 arguments: `req` (request) and `res` (response). The fir
 	- `name`: Header name
 	- `value`: Header value
 
-- `ondata`: value to setup handler function HTTP data. Handler function has 2 parameters:
+- `ondata`: value to setup handler function HTTP data. Handler function has 3 parameters:
 	- `self`: `req` object
 	- `chunk`: Request data
+	- `last`: set true while the last chunk
 
 The second object holds functions:
 
-- `send(self, data, [response_code])`: Function to send data to client. `self` is `req` object, `data` is data to send and `response_code` is HTTP response code like `200` or `404` (for example)
-- `send_header(self, header_name, header_data)`: Function to send HTTP headers to client. `self` is `req` object, `header_name` is HTTP header name and `header_data` is HTTP header data for client.
-- `finish([data])`: Function to finalize connection, optionally sending data. `data` is optional data to send on connection finalizing.
+- `send(data)`: Function to send data to client. `data` is data to send if the `data` is `nil` will close the connection
+and `response_code` is HTTP response code like `200` or `404` (for example)
```

Comment:
the response code line has to be removed. The status moved to the send_header method.

Reply:
OK